pkg/operator/resource_reconciler.go

```diff
@@ -357,12 +386,26 @@ func (rr *ResourceReconciler) onStatefulSetDelete(ss *appsv1.StatefulSet) {
 
 // EnqueueForReconciliation asks for reconciling the object.
 func (rr *ResourceReconciler) EnqueueForReconciliation(obj metav1.Object) {
-	rr.reconcileQ.Add(obj.GetNamespace() + "/" + obj.GetName())
+	level.Debug(rr.logger).Log(
+		"msg", "mariofar onadd",
+		"object", fmt.Sprintf("%s/%s", obj.GetNamespace(), obj.GetName()),
+		"object", fmt.Sprintf("%s/%s", obj.GetGenerateName(), rr.resourceKind),
+	)
+
+	if rr.hasOwnership(obj) {
+		rr.reconcileQ.Add(obj.GetNamespace() + "/" + obj.GetName())
+	} else {
+		level.Error(rr.logger).Log("msg", "%s object has been created by another prometheus operator, controllerID does not match, please be careful deploying multiple operators", rr.resourceKind)
```

Comment:
If we want a log message, it needs to be at debug level. Also moving to hasOwnership() is probably better.

Reply:
done